clients/java/src/test/java/io/camunda/zeebe/client/decision/SearchDecisionRequirementsTest.java

```diff
@@ -59,31 +59,27 @@ void shouldSearchDecisionRequirementsByKey() {
   @Test
   void shouldSearchDecisionRequirementsByResourceName() {
     // when
-    client
-        .newDecisionRequirementsQuery()
-        .filter(f -> f.dmnDecisionRequirementsName("resourceName"))
-        .send()
-        .join();
+    client.newDecisionRequirementsQuery().filter(f -> f.name("resourceName")).send().join();
 
     // then
     final DecisionRequirementsSearchQueryRequest request =
         gatewayService.getLastRequest(DecisionRequirementsSearchQueryRequest.class);
-    assertThat(request.getFilter().getDecisionRequirementsName()).isEqualTo("resourceName");
+    assertThat(request.getFilter().getName()).isEqualTo("resourceName");
```

Comment:
do you mean it should be `get{{Resource}}Name()` ? 
According to the naming conventions
```
* key and id fields should contain the entity as a prefix, e.g. userTaskKey, bpmnProcessId. This applies when referencing other resources, e.g. formKey in the user task entity. This applies in the respective entities themselves, too, e.g. userTaskKey in the user task entity.
* Other attributes of entities themselves should go without a prefix to avoid clutter, e.g. version in the process definition entity. In other resources, they have to be referenced with a prefix, e.g. processDefinitionVersion in the process instance entity
```
so `name` should be kept without the prefix because it is a direct attribute of the DecisionRequirements

Reply:
I think I was confused due to the literal "resourceName" because we have in [DecisionRequirementsItem](https://github.com/camunda/camunda/blob/18d34d3609bf97171d85c7895d7075055c4d7c04/zeebe/gateway-rest/target/generated-sources/openapi/src/main/io/camunda/zeebe/gateway/protocol/rest/DecisionRequirementsItem.java#L32-L32) a field with name `resourceName`. 

Can we use then a different literal for the value to avoid the confusion :) ?
This is then also valid for the other comments.

Reply:
Done @ralfpuchert